app/views/static/generic_object/custom-image-component.html.haml

```diff
@@ -19,7 +19,7 @@
                   :title     => t,
                   :alt       => t,
                   :enabled   => "true",
-                  'on-click' => "vm.uploadClicked(angularForm)"}
+                  'on-click' => "vm.uploadClicked()"}
```

Comment:
Can't really call this a bug, since technically, nothing is broken w.r.t `vm.uploadClicked` right now, or nothing is getting fixed with this change.